backend/oasst_backend/utils/hugging_face.py

```diff
@@ -8,10 +8,14 @@
 
 
 class HfUrl(str, Enum):
-    HUGGINGFACE_TOXIC_ROBERTA = ("https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta",)
+    HUGGINGFACE_TOXIC_ROBERTA = "https://api-inference.huggingface.co/models/unitary"
     HUGGINGFACE_FEATURE_EXTRACTION = "https://api-inference.huggingface.co/pipeline/feature-extraction"
 
 
+class HfClassificationModel(str, Enum):
+    TOXIC_ROBERTA = "multilingual-toxic-xlm-roberta"
```

Comment:
if you look at the hub, the model is called "unitary/multilingual-toxic-xlm-roberta", right now the "unitary" is in the URL part above and it should be here.